### ISSUE TITLE
NVDA cannot change or do anything with One Drive while syncing

### ISSUE THREAD

**Issue by KevanGP** [nvaccess/nvda#12522]
NVDA does not work properly with One Drive when syncing, cannot do anything because the recent files list keeps changing and NVDA keeps jumping to it whenever I tab to settings, before I can activate the settings button.

**Comment by KevGP** [comment]
One solution is to turn off my computer's Internet access when I want to change One Drive settings, close One Drive, or do anything else. But that's hardly a proper solution. It works but it's tedious.

**Comment by XLTechie** [comment]
What if you turn off "report dynamic content changes" (nvda+5)?

**Comment by Adriani90** [comment]
This is not reproducible in NVDA 2024.4.1 and last Onedrive version Build 24.211.1020.0001 (64-Bit).
Closing as works for me. if you are still having this issue, please comment and we can reopen.
